api/client/stack/deploy.go

```diff
@@ -106,7 +106,7 @@ func updateNetworks(
 		Labels: getStackLabels(namespace, nil),
 		Driver: defaultNetworkDriver,
 		// TODO: remove when engine-api uses omitempty for IPAM
-		IPAM: network.IPAM{Driver: "default"},
+		IPAM: &network.IPAM{Driver: "default"},
```

Comment:
This line could be removed as the engine-api change will allow omitting the IPAM.